public/static/docs/user-guide/dvcignore.md

```diff
@@ -44,9 +44,9 @@ it raises an error. Ignoring files inside such directory should be handled from
 
 The same as for [`.gitignore`](https://git-scm.com/docs/gitignore).
 
-## Example: Modification of ignored data
+## Example: Ignoring specific files
 
-Let's see what happens when we modify ignored file.
+Let's see what happens when we add a file to `.dvcignore`.
```

Comment:
`add and then try to modify` ....

Reply:
Oooooh I see what they meant with the previous subtitle now... That was confusing. Let me think about this ⏳

Reply:
I'm not sure about this suggestion. If you keep reading below, there's this line:

> Now, let's modify file `data1` and see if it affects `dvc status`.

so that covers what you were trying to include here, I think. Especially now that the example's subtitle has been removed.